src/components/Forms/SignupForm.jsx

```diff
@@ -0,0 +1,172 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  GridItem,
+  Heading,
+  Input,
+  SimpleGrid,
+  useColorModeValue,
+  Link,
+  VStack,
+  Spinner,
+  FormErrorMessage,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+import { Link as RouterLink, useLocation, Navigate } from 'react-router-dom';
+import { inputChangeHandler } from '../../utilities';
```

Comment:
can use absolute path https://stackoverflow.com/questions/41993461/import-react-components-with-absolute-path

Reply:
@Rohitprasad83 Thanks for the suggestion 🙌🏻